feat: add GitHub page with developer information

alessandrordgs opened the pull request:
Criei uma página que exibe quem está contribuindo para o projeto

alessandrordgs committed:
feat: add GitHub page with developer information

lumamontes commented on src/pages/github.astro:
```diff
@@ -0,0 +1,267 @@
+import Layout from "../layouts/Layout.astro"
+import TerminalWindow from "../components/TerminalWindow.astro"
+import TerminalLine from "../components/TerminalLine.astro"
+const devs = [
+  {
+    name: "Alessandro Rodrigues",
+    github: "https://github.com/alessandrordgs",
+  },
+  {
+    name: "Luma montes",
+    github: "https://github.com/lumamontes",
+  },
+]
```
aqui seria legal ficar em um arquivo separado, pode ser na raiz mesmo, tipo devs.json ou algo do tipo

alessandrordgs replied:
Coloquei um arquivo separado .ts mesmo pra ter tipagem

lumamontes commented on src/pages/github.astro:
```diff
@@ -0,0 +1,267 @@
+        <ul class="list-dash">
+          {
+            devs.map((dev) => (
+              <li class="underline-text">
+                <a href={dev.github} target="_blank" rel="noopener noreferrer">
+                  {dev.name}
+                </a>
+              </li>
```
ficou bem legall!! aqui só poderia ter a fotinha da pessoa, pegar pelo avatar do github mesmo

alessandrordgs replied:
Adicionei!!

lumamontes commented on src/pages/github.astro:
```diff
@@ -0,0 +1,267 @@
+        </a>
+      </section>
+
+      <!-- <TerminalLine prompt="dev@amapa:~$" command="" blinkingCursor={true} /> -->
```
```suggestion
```

lumamontes reviewed (commented):
Alguns pitacossss

alessandrordgs committed:
feat(devs): add amapa developers list and rename page

- Add devs-amapa.ts with developer profiles
- Rename github to devs page
- Include avatar display

lumamontes committed:
Merge branch 'main' into feat/github-page
